core/transfer/src/main/java/org/eclipse/dataspaceconnector/transfer/core/transfer/TransferProcessManagerImpl.java

```diff
@@ -156,6 +156,50 @@ public void enqueueCommand(TransferProcessCommand command) {
         commandQueue.enqueue(command);
     }
 
+    @Override
+    public void handleProvisionResult(String processId, List<ProvisionResponse> responses) {
+        var transferProcess = transferProcessStore.find(processId);
+        if (transferProcess == null) {
+            monitor.severe("TransferProcessManager: no TransferProcess found for deprovisioned resources");
```

Comment:
@jimmarino If I understand the changes in this PR correctly, this method handles the provisioning not the deprovisioning. If so the `deprovisioned` is wrong in this line or not?